src/app/users/users-achievements/users-achievements-update.scss

```diff
@@ -12,6 +12,11 @@
     height: inherit;
     padding: 0.25rem 0;
     white-space: pre-wrap;
+    margin-bottom: 0.25rem;
+  }
+
+  planet-step-list .mat-list-base .mat-list-item:last-child {
+    padding: 0.25rem;
```

Comment:
There is already top & bottom padding on `planet-step-list .mat-list-base .mat-list-item`.  All this does is add left & right padding to the last `.mat-list-item`, which creates a horizontal toolbar (see image).  Please remove this style.

![image](https://user-images.githubusercontent.com/9203229/52589715-7e4def00-2e0d-11e9-96df-cb06ac0bac4b.png)